docs/pom.xml

```diff
@@ -12,6 +12,16 @@
 	<artifactId>spring-cloud-gcp-docs</artifactId>
     <packaging>pom</packaging>
     <name>Spring Cloud GCP Documentation</name>
+	<dependencies>
```

Comment:
Can you fix the formatting in this file? looks like it was half indented with tabs, half with spaces.